docs/12-ContinuousIntegration.md

```diff
@@ -119,8 +119,28 @@ Travis doesn't provide visualization for XML or HTML reports so you can't view r
 
 ## GitLab
 
-[Work In Progress]
+If a file ```.gitlab-ci.yml``` exists in the root of the git repository, gitlab will run a pipeline each time you push to the gitlab server. The file configures the docker image that will be called. Below is a sample which loads a php7 docker image, clones your files, installs composer dependencies, runs the built-in php webserver and finally runs codeception:
+
+```
+image: mychiara/php7
```

Comment:
Also git needs to be installed and therefore also: git-man less libcurl3-gnutls liberror-perl libpopt0 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxext6 libxmuu1 openssh-client patch rsync xauth

The minimal .gitlab-ci.yml using an official image will be:
```yaml
# Select image from https://hub.docker.com/_/php/
image: php:7.0-apache

# Select what we should cache
cache:
  paths:
  - vendor/

before_script:
# Install git and unzip (composer will need them)
- apt-get update && apt-get install -qqy git unzip
# Install composer
- curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
# Install all project dependencies
- composer install
# Run webserver
- php -S localhost:8085 --docroot public &>/dev/null&

# Test
test:
  script:
  - vendor/bin/codecept run
```
This version will take around a minute longer to execute, then the one in the pull request.
Do you still prefer to use the official docker image?